pgcrypto/forms.py

```diff
@@ -0,0 +1,27 @@
+from django import forms
+
+
+class DateField(forms.DateField):
+    """
+    Form class for DateField.
+
+    Removes max_length since the backend model field is a text field.
+    """
+
+    def __init__(self, input_formats=None, *args, **kwargs):
+        """Init that pops off the max_length attribute."""
+        kwargs.pop('max_length', None)
+        super(DateField, self).__init__(input_formats, *args, **kwargs)
```

Comment:
This is python 3, you can remove the argument passed to self they would be filled by the compiler atomatically see https://docs.python.org/3/library/functions.html#super

Same comment apply for the `DateTimeField` class below.